packages/experimental/Switch/SPEC.md

```diff
@@ -88,7 +88,7 @@ export interface SwitchProps extends Omit<IWithPressableOptions<ViewProps>, 'onP
   onText?: string;
 
   /**
-   * Sets the position of the Switch's label
+   * Sets the position of the Switch's label. The position value 'after' is mutually exclusive with the onText and offText props.
```

Comment:
I'd also do a quick explanation for why they're mutually exclusive.

For example:
" The position value 'after' is mutually exclusive with the onText and offText props. This is due to the variable width of the text props negatively affecting the render of the Switch"

Reply:
Done, thanks!